bitnami/jenkins/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: jenkins
 sources:
   - https://github.com/bitnami/bitnami-docker-jenkins
   - https://jenkins.io/
-version: 10.1.1
+version: 10.1.2
```

Comment:
Adding new features should require a minor bump.

Reply:
Features are not new, they were already implemented but missing in the `values.yaml`. Not sure if this case counts as minor or patch.